docs/quick-start-guide.md

```diff
@@ -21,6 +21,8 @@ to build your own.
 
 ### Downloading a pre-built release:
 
+This method is not recommanded because those packages are now very old (0.21)
```

Comment:
Thank you!
Small typo:

recommanded -> recommended

Bonus: Remove the ` (0.21)` and put a period at the end of the sentence instead.